.github/workflows/clang_colcon.meta

```diff
@@ -0,0 +1,20 @@
+{
+    "names":
+    {
+        "fastrtps":
+        {
+            "cmake-args":
+            [
+                "-DFASTDDS_STATISTICS=ON",
+            ]
+        },
+        "fastdds_statistics_backend":
+        {
+            "cmake-args":
+            [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON",
```

Comment:
I'd remove this option from the test colcon.meta. Furthermore, I'd rename that other meta to something like test_colcon.meta, or even both of them to test.meta and clang.meta